code/LogFactory.php

```diff
@@ -12,9 +13,16 @@ class LogFactory implements Factory
     public function create($service, array $params = [])
     {
         $logger = new Logger('ss-oauth2');
-        $outputHandler = new StreamHandler(STDOUT);
-        $outputHandler->setFormatter(new LineFormatter("%level_name% - %message%\n"));
-        $logger->pushHandler($outputHandler);
+        $syslog = new SyslogHandler('ss-oauth2', LOG_AUTH, Logger::DEBUG);
```

Comment:
Hmm how about `SilverStripe_oauth2`? I think this is the convention with ss logs